pkgs/coverity/default.nix

```diff
@@ -0,0 +1,72 @@
+# SPDX-FileCopyrightText: 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  stdenv,
+  lib,
+  autoPatchelfHook,
+  fetchurl,
+  curl,
+  systemd,
+  zlib,
+  xorg,
+  alsaLib,
+  libxcrypt-legacy,
+  withUpdater ? true,
```

Comment:
I think we have no use for the `withUpdater` conditional

Reply:
Yes, removed and fixed.